src/runner/index.js

```diff
@@ -81,6 +85,7 @@ export default class Runner extends EventEmitter {
             await Promise.race(promises);
         }
         catch (err) {
+            await browserSet.reportTaskStatus(TASK_STATUS.errored, err);
```

Comment:
Taking in the account all the remarks above,  guess it's makes sense to move this functionality to the `BrowserJob` class